sources/packages/backend/apps/api/src/route-controllers/designation-agreement/designation-agreement.institutions.controller.ts

```diff
@@ -81,6 +86,13 @@ export class DesignationAgreementInstitutionsController extends BaseController {
         "User does not have the rights to create a designation agreement.",
       );
     }
+
+    // Check if institution is private or public and append it to the payload
+    payload.isBCPrivate =
```

Comment:
I think that the comment sounds like there are only those 2 types: public and private but we're checking if it is "BC Private", right?  It can be a private "International" institution.

Reply:
AFAIK, it is private vs "other types". Updated my comment line in the code